CMakeLists.txt

```diff
@@ -61,6 +61,7 @@ option(QL_USE_STD_SHARED_PTR "Use standard smart pointers instead of Boost ones"
 option(QL_USE_STD_UNIQUE_PTR "Use std::unique_ptr instead of std::auto_ptr" ON)
 option(QL_USE_STD_FUNCTION "Use std::function and std::bind instead of Boost ones" OFF)
 option(QL_USE_STD_TUPLE "Use std::tuple instead of boost::tuple" OFF)
+option(QL_MSVC_DISABLE_PARALLEL_BUILDS "Disable the /MP flag for parallel builds using msbuild" OFF)
```

Comment:
Should not be a package option.   Parallel building is a control feature used by the builder.  Don't hard code an option for this.